pkg_resources/__init__.py

```diff
@@ -2680,7 +2680,7 @@ def parse_map(
             _data = data.items()
         else:
             _data = split_sections(data)
-        maps: Dict[str, Dict[str, "EntryPoint"]] = {}
+        maps: Dict[str, Dict[str, EntryPoint]] = {}
```

Comment:
Newer version of `mypy` complaining?

Reply:
Shouldn't we fix this in a separate PR or commit, since it's unrelated to the proposal?

Reply:
Agreed, I will work on it (this PR was more of a proof of concept).

Reply:
https://github.com/pypa/setuptools/pull/4393